variables.tf

```diff
@@ -74,68 +94,110 @@ variable squid_config {
   default     = ""
 }
 
+variable metadata {
+  description = "Metadata to be attached to the NAT gateway instance"
+  type        = "map"
+  default     = {}
+}
+
+variable ssh_source_ranges {
+  description = "Network ranges to allow SSH from"
+  type        = "list"
+  default     = ["0.0.0.0/0"]
+}
+
+variable instance_labels {
+  description = "Labels added to instances."
+  type        = "map"
+  default     = {}
+}
+
+variable service_account_email {
+  description = "The email of the service account for the instance template."
+  default     = "default"
+}
+
+variable autohealing_enabled {
+  description = "Enable instance autohealing using http health check"
+  default     = false
+}
+
 variable region_params {
   description = "Map of default zones and IPs for each region. Can be overridden using the `zone` and `ip` variables."
   type        = "map"
 
   default = {
-    us-west1 {
-      zone = "us-west1-b"
+    asia-east1 {
```

Comment:
The `=` assignment operator is conspicuously missing from this one. I see it's the same in upstream though, but I would say it should be there.